app/src/main/java/com/keylesspalace/tusky/adapter/PlaceholderViewHolder.kt

```diff
@@ -29,19 +29,19 @@ import com.keylesspalace.tusky.util.visible
  */
 class PlaceholderViewHolder(
     private val binding: ItemStatusPlaceholderBinding,
-    private val listener: StatusActionListener
+    listener: StatusActionListener
 ) : RecyclerView.ViewHolder(binding.root) {
 
+    init {
+        binding.loadMoreButton.setOnClickListener {
+            binding.loadMoreButton.hide()
+            binding.loadMoreProgressBar.show()
+            listener.onLoadMore(bindingAdapterPosition)
+        }
```

Comment:
I think this change is causing a bunch of unneeded "show more" buttons in my notifications for non-sensitive posts